src/main/java/org/opentripplanner/routing/core/State.java

```diff
@@ -50,6 +50,12 @@
     public static Collection<State> getStates(RoutingRequest request) {
```

Comment:
This is not something you have changed, but I realize `getInitialStates` is probably a better method name here.

Reply:
Good idea, the method was renamed.